python/shotgun_model/shotgun_model.py

```diff
@@ -440,6 +440,7 @@ def _load_data(
                                           each column in the row contains the value for the corresponding field from columns. This means
                                           that the data from the loaded entity will be available field by field. Subclasses can modify
                                           this behavior by overriding _get_additional_columns.
+        :param editable_columns:          A subset of ``columns`` that will be editable in views that use this model.
```

Comment:
Can i make a model which only has got a single column (e.g. like the left hand side loader treeview) but where that column has editable values? Another use case that could be great to cover via a usage example, either as part of the parameter or in the main docstring. A lot of TDs (and developers!) are recipe driven, the more examples we provide, the quicker they will pick up the concepts i think.